src/frontend/packages/lib_video/src/utils/localstorage.ts

```diff
@@ -3,7 +3,7 @@ import { v4 as uuidv4 } from 'uuid';
 
 export const ANONYMOUS_ID_KEY = 'marsha_anonymous_id';
 
-const localStorageEnabled = (): boolean => {
+export const localStorageEnabled = (): boolean => {
```

Comment:
Yeah probably lib-component since lib-common is mainly for code shared beetween tests (lib-tests) and the other libs (so yes common is a pretty poor name)